### ISSUE TITLE
Show spaceman-diff as a comment in a pull request

### ISSUE THREAD

**Issue by MichaelDimmitt** [holman/spaceman-diff#12]
Show spaceman-diff as a comment in a pull request

Im going to try to get this working. 
If colors are not achievable I will try to send the comment in black and white.

Simular to how [codecov](https://github.com/codeforboston/cliff-effects/pull/643) is executed by travis-ci and submits a comment on a pr.

example: 
```bash
svn export https://github.com/MichaelDimmitt/spacediffs-for-apple-natgeo-images-/trunk/spacediff.md; 
cat spacediff.md;
```
In my example I made an md file. It could have any extension. 
But spaceman-diff could write these files to a hidden folder called .spaceman-diffs
and then in a travis.ci or circle.ci when a pr occurs, a program could look for that folder cat the result and submit a comment to github or gitlab.

What do you think? Im okay with others working on this. 
I will see how far I can get and hopefully I do not get stuck.

**Comment by holman** [comment]
Yeah, that sounds pretty cool! It's not something I'd personally use too much, I think, since you have the actual image diff not too far away in this pull, but happy to help out somehow if I can!

**Comment by MichaelDimmitt** [comment]
@holman, I got the diff working for a git project!
I stripped the colors out. So it is a colorless diff.

Make sure your diff setting is on unified view not split view.

[NatGeo02.jpg](https://github.com/MichaelDimmitt/spacediffs-for-apple-natgeo-images-/commit/a355daa108ed650dac47ff274ab65a77a51578de#diff-c396e848d55150a1267a2191b582669aR34) - is a polar bear swimming underwater in this commit.
[NatGeo11.jpg](https://github.com/MichaelDimmitt/spacediffs-for-apple-natgeo-images-/commit/a355daa108ed650dac47ff274ab65a77a51578de#diff-c396e848d55150a1267a2191b582669aR322)  - is a tree

You have to scroll right to see them. They are in the "NEW" section.

https://github.com/MichaelDimmitt/spacediffs-for-apple-natgeo-images-/commit/a355daa108ed650dac47ff274ab65a77a51578de

Anyway Ill keep working on it, this space-man diff repo is pretty cool.

**Comment by holman** [comment]
Hah, that's cool! Would be nice to be able to pass a width or something into spaceman-diff to reduce the size when needed. But in general that's pretty sweet.

**Comment by MichaelDimmitt** [comment]
@holman, I agree! Right now I am restricting width of 2 with bash.
I could handle the width on my end with the external program.

But do you think it should be an internal feature of your app or handled as an external program.
I am fine with whichever decision.
